src/filter.h

```diff
@@ -36,6 +36,7 @@ extern void quisk_filt_cfInit(struct quisk_cfFilter *, float *, int);
 extern void quisk_filt_destroy(struct quisk_cfFilter *);
 extern void quisk_cfTune(struct quisk_cfFilter *, float);
 extern void quisk_ccfFilter(complex float *, complex float *, int, struct quisk_cfFilter *);
+extern void quisk_ccfFilter_realonly(complex float *, complex float *, int, struct quisk_cfFilter *);
 
```

Comment:
How about using the Quisk nomenclature, I think `cc` means complex in complex out, so the function could be named: `quisk_crfFilter`.

Reply:
Works for me. I'll rename it.